tests/resolvers/Mutation/createFamilyGroup.spec.ts

```diff
@@ -0,0 +1,76 @@
+import "dotenv/config";
+import type mongoose from "mongoose";
+import { Types } from "mongoose";
+import type { MutationCreateFamilyGroupArgs } from "../../../src/types/generatedGraphQLTypes";
```

Comment:
You have interchangeably used the term `familyGroup` and `Family` in the PR definitions. To make it unambiguous for future developers use `userFamily`  to replace `familyGroup` and `Family`.

Reply:
done